docs/blender_docs/io_gltf2.rst

```diff
@@ -30,7 +30,198 @@ This importer/exporter supports the following glTF 2.0 features:
 - Animation (keyframe, shape key, and skinning)
 
 
-.. rubric:: Supported
+Meshes
+------
+
+glTF's internal structure mimics the memory buffers commonly used by graphics chips
+when rendering in real-time, such that assets can be delivered to desktop, web, or mobile
+clients and be promptly displayed with minimal processing.  As a result, quads and N-gons
+(faces with more than 3 edges) are automatically converted to triangles when
+saving to glTF.  Likewise, curves and and other non-mesh data are not preserved,
+and should be converted to meshes prior to export.
+
+Materials
+---------
+
+The core material system in glTF supports a metal/rough PBR workflow with the following
+channels of information:
+
+- Base Color
+- Metallic
+- Roughness
+- Baked Ambient Occlusion
+- Normal Map
+- Emissive
+
+.. note::
+
+   When images are used to drive these channels, glTF requires that PNG or JPG format
+   is used to pack the images.
+
+.. note::
+
+   glTF does not directly contain Blender's nodes, however, this glTF exporter
+   and importer will use nodes to describe the contents of the glTF file.
```

Comment:
I'm not sure what this note is meant to convey – do you mean this _manual_ will use nodes to describe the contents? Users can work in the Blender material sidebar without opening the node graph now, it generates a parallel node graph internally.

Reply:
Deleted.